authors/apps/authentication/tests/test_users.py

```diff
@@ -105,31 +117,42 @@ def test_update_with_user_details_exist(self):
         """
         Test user update with user details that exist
         """
-        self.is_authenticated("admin@authors.com", "adm123Pass!!")
+        logdata = self.login_user("admin@authors.com", "adm123Pass!!")
+        token = logdata.data['token']
+        self.client.credentials(
+            HTTP_AUTHORIZATION='Bearer ' + token)
```

Comment:
@IssaIan This block could be put in one method to implement the DRY principle and the call it in the tests required
for instance, you could authentication method that you could call in one line

Reply:
@a-braham I have implemented that Thanks for the feedback.